### ISSUE TITLE
Tracking issue: reorganise sections in all manuals

### ISSUE THREAD

**Issue by DanielSidhion** [DanielSidhion/fictional-eureka#1]
Depends on:
- tool to ensure every possible link on the manual can still be reached after every change.
- (optional) standard for anchor links.

This is a tracking issue for reorganising sections in all manuals.
This covers 3 kinds of reorganisation:
- Moving content from a manual to another where the content has a better fit.
- Deduplicating content between manuals when there is overlap.
- Creating new sections, removing existing ones, and shuffling content within each manual to make information easier to find.

The manuals have a lot of content, and sometimes they can benefit from a focused effort to look at the sections in all manuals at a high level and organise them.
At the same time, content in the manuals are always getting updated, which means as new content gets added, a better way to organise a manual will show up.

Because of this, this tracking issue is organised in three steps:
1. Focused effort to look at all sections from manuals from a high level.
  - Once this effort is done, we'll know which sections can be changed immediately to benefit our readers.
2. Gather opportunities to improve sections as efforts to update content in the manuals progress.
  - We'll keep a list of sections that are harder to understand and ideas to make them better.
3. Act when we have a better idea of a new section organisation which improves the status quo.
  - We'll remove the sections that get "fixed" through this effort from the list of opportunities we're tracking.

Step 1 is a one-time effort that has to be done periodically.
We have a workflow to open a new issue on a schedule to remind the team to do that work.

Steps 2 and 3 are intended to be done continuously.
We have a workflow to ensure an open issue always exists to keep track of the list of sections that could be improved.
Once an existing open issue gets too long or old, it's ok to close it and open a new one.
Please move the list from the old issue to the new one when this is done.

Since this is related to a maintenance task, this tracking issue is intended to always be open.
Please do not comment on this issue.
Comments will be deleted to keep the issue as clean as possible, since it'll remain open for a long time.
If you have comments on it, please open a new issue where the discussion can take place.
